### ISSUE TITLE
Skip death, delay reduction

### ISSUE THREAD

**Issue by TheReVeaLz** [MathewSachin/Fate-Grand-Automata#345]
can you add this to Battle.kt?
remove NP delay after click Battle also make fine tune for performBattle() seconds wait
#skipDeathAnimation loc is 1700, 100

fun performBattle() {
        screenshotManager.useSameSnapIn { onTurnStarted() }
        1.5.seconds.wait()
        ...................
        card.clickCommandCards()
        10.seconds.wait()
        if (prefs.skipDeathAnimation == true) {
            game.skipDeathAnimation.click(300)
        }
    }

and make adjusment for finetune cap
<kbd><img src="https://user-images.githubusercontent.com/70447633/91757320-4f1def00-ebf8-11ea-8b6b-17173230aa6e.png" width="350"></kbd>

**Comment by MathewSachin** [comment]
Skip enemy death or Servant (Arash/Chen gong sacrifice) death?
For Enemy death, there's already an option in-game.

I think it's better for the servant death animation to finish by itself instead of waiting for 10 seconds and then clicking 300 times.
Even if those 10 seconds are configurable in a fine-tune setting, servants don't die every turn. You'd save more time not worrying about this (Unless there's a proper way to detect servant death).
I've already denied the request here: #185

Honestly, you should've listed the caps you wanted to be changed instead of making me go through all of them.
I'll tweak a few which I think would be reasonable.

**Comment by TheReVeaLz** [comment]
yes for chen/arash, maybe u right but for 1.5 seconds delay after click attack + 1.5 seconds if using np that's too much delay for waiting card to show up. I ran perfectly with just 1.5 seconds delay after click attack without extra 1.5 seconds if using NP maybe u can just make it with fine tune for user

Finetune wait after click lowest cap to 20ms and highest 1500ms (i think no one will use it at max delay tho)

**Comment by MathewSachin** [comment]
>  but for 1.5 seconds delay after click attack + 1.5 seconds if using np that's too much delay for waiting card to show up. I ran perfectly with just 1.5 seconds delay after click attack without extra 1.5 seconds if using NP maybe u can just make it with fine tune for user

Are you looking at the latest code. I already changed that 1.5 + 1.5 = 3s to 2s.
The brave chains feature relies on NP cards being loaded before detection so in that case it is even more important to have some delay.
I like to avoid having more fine-tune settings unless absolutely necessary. Those settings are easy to mess up for users 🤷‍♂️ 

https://github.com/MathewSachin/Fate-Grand-Automata/blob/42b88e1be7ed2dd123c603c14dd65abbe27341af/scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/modules/Battle.kt#L43-L55

> Finetune wait after click lowest cap to 20ms

I made the lowest 0ms for those who live on the edge (don't recommend though). Leaving the highest as it is.
